core/blockly.js

```
@@ -123,6 +126,49 @@ Blockly.hueToRgb = function(hue) {
+
+/**
+ * converts hsva to rgba
+ * @param {Number} hue the color
+ * @param {Number} saturation the saturation
+ * @param {Number} value the brightness
+ * @param {Number} alpha the transparency
+ * @returns {Number} the hex color
+ */
+goog.color.hsvaToHex = function(hue, saturation, value, alpha) {
+  var hex = goog.color.hsvToHex(hue, saturation, value)
+  var alpha = goog.constrain(Math.floor(alpha * 255), 0, 255).toString(16)
```

Biome (2.1.2)

[error] Shouldn't redeclare 'alpha'. Consider to delete it or rename it.

'alpha' is defined here:


(lint/suspicious/noRedeclare)